modules/ztp-setting-managed-bare-metal-host-kernel-parameters-manual-workflow_gitops_workflow.adoc

```diff
@@ -0,0 +1,97 @@
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+
+.Procedure
+
+. Create the `InfraEnv` CR and edit the `spec.kernelArguments` specification to configure kernel parameters.
+
+.. Save the following YAML in an `InfraEnv-example.yaml` file:
++
+[NOTE]
+====
+The `InfraEnv` CR has template variables such as `{{ .Cluster.ClusterName }}`. The `SiteConfig` CR automatically populates values for these templates during deployment. Do not edit the templates manually.  
+====
++
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
+  name: "{{ .Cluster.ClusterName }}"
+  namespace: "{{ .Cluster.ClusterName }}"
+spec:
+  clusterRef:
+    name: "{{ .Cluster.ClusterName }}"
+    namespace: "{{ .Cluster.ClusterName }}"
+  kernelArguments:
+    - operation: append <1>
+      value: audit=0 <2>
+    - operation: append 
+      value: trace=1
+  sshAuthorizedKey: "{{ .Site.SshPublicKey }}"
+  proxy: "{{ .Cluster.ProxySettings }}"
+  pullSecretRef:
+    name: "{{ .Site.PullSecretRef.Name }}"
+  ignitionConfigOverride: "{{ .Cluster.IgnitionConfigOverride }}"
+  nmStateConfigLabelSelector:
+    matchLabels:
+      nmstate-label: "{{ .Cluster.ClusterName }}"
+  additionalNTPSources: "{{ .Cluster.AdditionalNTPSources }}" 
+----
+<1> Specify the append operation to add a kernel argument.
+<2> Specify the kernel argument you want to configure. This example configures the audit kernel parameter and the trace kernel parameter. 
+
+. Commit the `InfraEnv-example.yaml` CR to the same location in your Git repository that has the `SiteConfig` CR and push your changes.
+
+. Edit the `spec.clusters.crTemplates` specification in the `SiteConfig` CR to reference the `InfraEnv-example.yaml` CR in your Git repository:
++
+[source,yaml,options="nowrap",role="white-space-pre"]
+----
+clusters:
+  crTemplates:
+    InfraEnv: "InfraEnv-example.yaml"
+----
++
+When you are ready to deploy your cluster by committing and pushing the `SiteConfig` CR, the build pipeline uses the custom `InfraEnv-example` CR in your Git repository to configure the infrastructure environment, including the custom kernel parameters. 
+
+.Verification
+To verify that the kernel parameters are applied to the managed bare-metal host, enter a debug session on the target host and use the `sysctl` command to view the kernel parameters:
+
+. Enter into a debug session on the target node: 
++
+[source,bash,options="nowrap",role="white-space-pre"]
+----
+$ oc debug node/<cluster_node>
+----
+
+. Set `/host` as the root directory within the debug shell prompt:
++
+[source,bash,options="nowrap",role="white-space-pre"]
+----
+# chroot /host
+----
+
+. Search the system's kernel parameters by using the following command:
++
+[source,bash,options="nowrap",role="white-space-pre"]
+----
+sysctl -a | grep <kernal_parameter>
```

Comment:
Are you sure it is the right way to verify it? I verified by the contents of **/proc/cmdline**